torch/csrc/jit/ir.h

```diff
@@ -515,8 +524,10 @@ friend struct Node;
     return output_;
   }
 
-  Node * addInput() {
-    return addInput(create(kParam));
+  Node * addInput(std::string name_="") {
```

Comment:
nit: why the underscore in argument name? It makes it easier to confuse with object attributes

Reply:
I think in some intermediate state of this diff this was a constructor argument, and that seems to be the convention there (e.g. see Node::Node()).

I'll change it.